rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
```

Comment:
Just changed it to `"Documents\\GTA3 User Files"` for now.
Seams using `SHGetKnownFolderPath` means adding more includes and duplicating a bit more code? Having some issues (don't really know what I'm doing), so perhaps it'd be best left for someone else if you want it like that.